executor/join.go

```diff
@@ -405,6 +411,15 @@ func (e *HashJoinExec) waitJoinWorkersAndCloseResultChan() {
 }
 
 func (e *HashJoinExec) runJoinWorker(workerID uint, probeKeyColIdx []int) {
+	probeTime := int64(0)
+	if e.stats != nil {
+		start := time.Now()
+		defer func() {
+			atomic.AddInt64(&e.stats.probe, probeTime)
```

Comment:
emm.... accumulated probe time seems useless, how about dividing it by total rows to get the average probe time?

Reply:
Or record it for each worker separately.

Reply:
No, The total time can use to indicate the total time consuming of the `CPU_TIME`.